x-pack/plugins/security_solution/public/overview/components/detection_response/cases_by_status/use_cases_by_status.tsx

```diff
@@ -28,8 +29,9 @@ export const useCasesByStatus = ({ skip = false }) => {
   const {
     services: { cases },
   } = useKibana();
-  const { to, from } = useGlobalTime();
-
+  const { to, from, setQuery, deleteQuery } = useGlobalTime();
+  // create a unique, but stable (across re-renders) query id
```

Comment:
Was curious, why does this need to be unique? In other places I think Ive seen it hard coded.

Reply:
If two components are on the same page and call `useCasesByStatus` they would have the id. It would register the query as being the same even though they could be using a different search parameter.

Reply:
oh that makes sense. Thank you